docs/src/operate/security/certificate-signing-request.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Certificate signing request
+tags: [Operate, Security, Cloud]
+description: Generate certificate signing request for %%te%%
+---
+
+If you want to use a device certificate which is signed by a Certificate Authority (CA), you can generate the Certificate Signing Request (CSR), which is later used by CA to generate a device certificate. This process requires additional tooling as %%te%% provides you only with the CSR. 
+
+## Create a certificate signing request
+
+To create a CSR you can use [`tedge cert create-csr`](../../references/cli/tedge-cert.md) %%te%% command:
+
+```sh
+sudo tedge cert create-csr --device-id alpha
+```
+
+or
+
+```sh
+sudo tedge cert create-csr
+```
+if a self-signed certificate already exists and you want to reuse device name.
```

Comment:
Technically it does not have to be a self-signed certificate, just any existing device certificate.

```suggestion
if a device certificate already exists and you want to reuse device name.
```